frontend/apps/marketing/src/config/doubleTheDonation.ts

```diff
@@ -0,0 +1,4 @@
+// The public API key for Double the Donation
+// Used for the Double the Donation integration on code.org/donate
+// See: https://support.doublethedonation.com/knowledge/getting-started-understanding-api-keys
+export const DOUBLE_THE_DONATION_PUBLIC_KEY = '4RoxGGsUoWBRMUfj';
```

Comment:
Seems to be working:

<img width="1617" alt="Screenshot 2025-06-17 at 3 30 41 PM" src="https://github.com/user-attachments/assets/b5090994-b5b2-47d8-8a63-0f3638bf4c7f" />